### PR TITLE
Divvier

### PR DESCRIPTION
**Pull request by emilhaegglund**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/divvier/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -eu -o pipefail
+
+echo ${PREFIX}
+mkdir -p ${PREFIX}/bin
+make
```
Comment (dpryan79) [inline recipes/divvier/build.sh]: You can use `make CC=$CC CPP=$CXX` and get rid of the patch

Comment (emilhaegglund) [reply recipes/divvier/build.sh]: Thanks for the help. Didn't know about this.

File: recipes/divvier/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    number: 0 
+
+requirements:
+    build:
+        - {{ compiler('cxx') }} 
+        - {{ compiler('c') }}
+    host:
+        - zlib
+    run:
+        - zlib
```
Comment (druvus) [inline recipes/divvier/meta.yaml]: I think zlib should be exporting itself so it could be removed from run section

**Comment by druvus** [comment]
thanks @emilhaegglund

**Pull request merged**